views/lti/mainNavbar.jade

```diff
@@ -32,9 +32,22 @@ div.main-navbar
             msg.className = 'empty-cke-warning';
             msg.innerHTML = 'Enter edit mode to fill in this section...';
 
-            const containerId = cke.container.getParent().getParent().getParent().getId();
-            document.querySelector('#' + containerId + ' form').appendChild(msg);
-            document.querySelector('#' + containerId + ' .cke').style.display = 'none';
+            let container = cke.container.getParent().getParent().getParent();
```

Comment:
This is because of different HTML structures between the 3 deliverables.
It tries to handle everything.
What it does is: find empty CKE and hide it and show a message to enter edit mode to be able to edit content.
Will need to refactor this by injecting the message into CKE.